<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoverControl</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_launch_rover_process">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Launch Rover Process</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ssh_connnection_status">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SSH Connection Status:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_software_connnection_status">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Software Connection Status:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reduced_comms">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Reduced Comms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_drive_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drive Status:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>781</width>
      <height>491</height>
     </rect>
    </property>
    <property name="title">
     <string>Map</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWebEngineView" name="map_widget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header>qwebengineview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
